src/Uno.UI/UI/Xaml/Shapes/Path.iOSmacOS.cs

```diff
@@ -11,15 +11,16 @@ public partial class Path : Shape
 
 		/// <inheritdoc />
 		protected override Size MeasureOverride(Size availableSize)
-			=> MeasureAbsoluteShape(availableSize, GetPath());
+			=> MeasureAbsoluteShape(availableSize, GetPathAndFillRule(out _));
 
 		/// <inheritdoc />
 		protected override Size ArrangeOverride(Size finalSize)
-			=> ArrangeAbsoluteShape(finalSize, GetPath());
+			=> ArrangeAbsoluteShape(finalSize, GetPathAndFillRule(out var fillRule), fillRule);
 
-		private CGPath GetPath()
+		private CGPath GetPathAndFillRule(out FillRule fillRule)
```

Comment:
What about returning a named Tuple instead?

Reply:
The goal was to use it fluently. With a Tuple it's not possible.